docs/index.md

```
@@ -1,90 +1,90 @@
 
-LLxprt Code brings the capabilities of large language models to your terminal in an interactive Read-Eval-Print Loop (REPL) environment. LLxprt Code consists of a client-side application (`packages/cli`) that communicates with a local server (`packages/core`), which in turn manages requests to the provider APIs and its AI models. LLxprt Code also contains a variety of tools for tasks such as performing file system operations, running shells, and web fetching, which are managed by `packages/core`.
+```bash
+npm install -g @vybestack/llxprt-code
+llxprt
+```
 
-## Navigating the documentation
+Once inside the REPL, authenticate with a provider and start coding:
 
-### Getting Started
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)